docs/src/guides/testing.md

```diff
@@ -46,6 +46,10 @@ View the inscription in the regtest explorer:
 ```
 ord -r server
 ```
+If you want to test Brotli encoded content over http you need to add:
```

Comment:
```suggestion

By default, browsers don't support compression over HTTPS. To test compressed content over HTTP, add the `--decompress` flag:
```

Reply:
we're only decompressing brotli so made it explicit